deploy/olm-catalog/ibm-iam-operator/3.6.1/operator.ibm.com_oidcclientwatchers_crd.yaml

```diff
@@ -0,0 +1,73 @@
+apiVersion: apiextensions.k8s.io/v1beta1
+kind: CustomResourceDefinition
+metadata:
+  name: oidcclientwatchers.operator.ibm.com
+  labels:
+    app.kubernetes.io/instance: ibm-iam-operator
+    app.kubernetes.io/managed-by: ibm-iam-operator
+    app.kubernetes.io/name: ibm-iam-operator
+spec:
+  group: operator.ibm.com
+  names:
+    kind: OIDCClientWatcher
+    listKind: OIDCClientWatcherList
+    plural: oidcclientwatchers
+    singular: oidcclientwatcher
+  scope: Namespaced
+  subresources:
+    status: {}
+  validation:
+    openAPIV3Schema:
+      description: OIDCClientWatcher is the Schema for the oidcclientwatchers API
```

Comment:
`Schema` lower case?

Reply:
I don't know why, but, the CRD is named like this `oidcclientwatchers.operator.ibm.com`, though we give the name of the type of oidcclientwatcher. I think `oidcclientwatchers` is all right.